lib-src/libsndfile/src/common.h

```diff
@@ -354,10 +354,9 @@ typedef struct
 	void 			*handle, *hsaved ;
 
 	int				use_wchar ;
-#else
+#endif
 	/* These fields can only be used in src/file_io.c. */
 	int 			filedes, savedes ;
```

Comment:
You make no use of savedes.  So the more minimal change logically (though not textually) should leave savedes out of conditional compilation for Windows.